tests/playbooks/roles/install-devstack/templates/local.conf.j2

```diff
@@ -0,0 +1,116 @@
+[[local|localrc]]
+RECLONE=False
+HOST_IP={{ local_ip_address }}
+DEST=/opt/stack
+DATA_DIR=${DEST}/data
+USE_PYTHON3=True
+LOGFILE=$DEST/logs/stack.sh.log
+VERBOSE=True
+LOG_COLOR=False
+LOGDAYS=1
+SERVICE_TIMEOUT=300
+
+DATABASE_PASSWORD=password
+ADMIN_PASSWORD=password
+SERVICE_PASSWORD=password
+SERVICE_TOKEN=password
+RABBIT_PASSWORD=password
+
+TARGET_BRANCH={{ branch }}
+
+ENABLED_SERVICES={{ enable_services | join(',') }}
+
+{% if "nova" in enable_services %}
+# Nova
+enable_plugin nova https://opendev.org/openstack/nova.git {{ branch }}
+enable_service n-api
+enable_service n-cpu
+enable_service n-cond
+enable_service n-sch
+enable_service n-api-meta
+
+enable_service placement-api
+enable_service placement-client
+{% endif %}
+
+{% if "glance" in enable_services %}
+# Glance
+enable_plugin glance https://opendev.org/openstack/glance.git {{ branch }}
+enable_service g-api
+enable_service g-reg
+
+[[post-config|$GLANCE_API_CONF]]
+[glance_store]
+default_store = file
+{% endif %}
+
+{% if "cinder" in enable_services %}
+# Cinder
+enable_plugin cinder https://opendev.org/openstack/cinder.git {{ branch }}
+enable_service cinder
+enable_service c-api
+enable_service c-vol
+enable_service c-sch
+{% endif %}
+
+{% if "neutron" in enable_services %}
+# Neutron
+enable_plugin neutron https://opendev.org/openstack/neutron.git {{ branch }}
+enable_service q-svc
+enable_service q-agt
+enable_service q-dhcp
+enable_service q-l3
+enable_service q-meta
+IP_VERSION=4
+IPV4_ADDRS_SAFE_TO_USE=10.1.0.0/26
+FIXED_RANGE=10.1.0.0/26
+NETWORK_GATEWAY=10.1.0.1
+FLOATING_RANGE=172.24.5.0/24
+PUBLIC_NETWORK_GATEWAY=172.24.5.1
+Q_PLUGIN=ml2
+Q_ML2_TENANT_NETWORK_TYPE=vxlan
+Q_DVR_MODE=legacy
+Q_AGENT=openvswitch
+Q_ML2_PLUGIN_MECHANISM_DRIVERS=openvswitch
```

Comment:
I knew OVN/geneve is default now, any special reason we stick to legacy ovs ?

Reply:
@lingxiankong?

Reply:
how about check
https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/1009/files
as reference? @gman0 @lingxiankong

Reply:
> geneve is default now, any special reason we stick to legacy ovs ?

Because we have been using devstack with OVS backed Neutron for a long time, it's working well, and I've never heard that OVS plugin is going to be deprecated from OpenStack community. Switching to OVN will cause us lots of pain.

Reply:
Ididn't say it's deprecate.. but default to ovn now and seems this is the future..